src/ArtistList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ArtistDetail from './ArtistDetail';
+import Artist from './Artist';
+
+class ArtistList extends React.Component{
+    state ={
+        sortType: '',
```

Comment:
Please use `null` as the empty state instead of empty strings when initializing state.